README.rst

```diff
@@ -246,24 +270,25 @@ Iterator over the classifier choices for a single symbol:
 
     with PyTessBaseAPI() as api:
         api.SetImageFile('/usr/src/tesseract/testing/phototest.tif')
-        api.SetVariable("save_blob_choices", "T")
```

Comment:
That is long gone. I am not sure what is needed for legacy models today. But for LSTMs, it's `lstm_choice_mode`. Unfortunately, it usually does not yield any actual results. It's complicated...